README.md

```
@@ -2,6 +2,7 @@
 
 # 🚀 TalentMesh
 
+<img src="public/images/talentmesh-banner.png" alt="TalentMesh Banner" width="100%">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: img

(MD033, no-inline-html)